roles/sideload_kernel/tasks/main.yml

```
@@ -0,0 +1,124 @@
+---
+- name: Validate required variables
+  ansible.builtin.assert:
+    that:
+      - sideload_kernel_uri | string
+
+- name: Install kubernetes.core dependencies
+  block:
+    - name: Prefer package manager
+      become: true
+      ansible.builtin.package:
+        name: python3-kubernetes
+        state: present
+  rescue:
+    - name: Fallback to pip installation
+      ansible.builtin.pip:
+        name: kubernetes
+
+- name: Sideload-kernel job namespace
+  kubernetes.core.k8s:
```

Please switch to community.kubernetes, we don't have support for kubernetse.core.k8s yet as newer versions only support newer versions of ansible and we support 2.9
```suggestion
  community.kubernetes.k8s:
```